content/docs/introduction/comparison.md

```diff
@@ -164,8 +163,12 @@ Where Prometheus is better:
 
 InfluxDB is maintained by a single commercial company following the open-core
 model, offering premium features like closed-source clustering, hosting and
-support. Prometheus is a [fully open source and independent project](/community/), maintained
-by a number of companies and individuals, some of whom also offer commercial
+support. For example, [Enterprise InfluxDB](https://docs.influxdata.com/enterprise_influxdb/v1.3/) 
```

Comment:
I think it's a little unfair to Influx to only mention the commercial Kapacitor down here, it makes it seem as though there's no commercial clustered option. I'd put this up in the Kapacitor section.

This section is more about how the projects are setup.

Reply:
Ok, I've added lines for "Enterprise Kapacitor" in the top and bottom sections now... Please review again.

Reply:
The changes to this paragraph should be removed.